.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Important considerations when opening a pull request:
+
+1. Make sure you do not make the changes you want open a pull request on the `master` branch of your fork, or open the pull request from the `master` branch of your fork. Some of our integrations will fail if you do this, resulting in your pull request not being accepted. If this is your first pull request, it is probably a good idea to first read up on how opening pull requests work (https://opensource.com/article/19/7/create-pull-request-github is a good introduction);
```

Comment:
"Make sure you do not make the changes you want open a pull request on the `master` branch of your fork, or open the pull request from the `master` branch of your fork.".